src/commands/synthetics/interfaces.ts

```diff
@@ -320,6 +320,7 @@ export interface LocationsMapping {
 export interface Trigger {
   batch_id: string
   locations: Location[]
+  selective_rerun_rate_limited?: boolean
```

Comment:
So does the backend return this information then? 🤔

Reply:
From the PR description:
> In that case, the backend now returns a `selective_rerun_rate_limited` property.

Yes, that's a backend PR that I did for the same Jira card